sapi/cli/config.m4

```diff
@@ -6,7 +6,7 @@ PHP_ARG_ENABLE(cli,,
 [  --disable-cli           Disable building CLI version of PHP
                           (this forces --without-pear)], yes, no)
 
-AC_CHECK_FUNCS(setproctitle)
+AC_CHECK_FUNCS(setproctitle setproctitle_fast)
```

Comment:
it should be in `sapi/fpm/config.m4` I guess...

Reply:
True I wanted to put the feature in cli too firstly but did not see the use afterwards...